crates/bevy_render/src/batching/gpu_preprocessing.rs

```diff
@@ -1122,10 +1122,10 @@ impl FromWorld for GpuPreprocessingSupport {
             // `max_compute_*` limits to zero, so we arbitrarily pick one as a canary.
             device.limits().max_compute_workgroup_storage_size != 0;
 
-        let downlevel_support = adapter.get_downlevel_capabilities().flags.contains(
-            DownlevelFlags::COMPUTE_SHADERS |
-            DownlevelFlags::VERTEX_AND_INSTANCE_INDEX_RESPECTS_RESPECTIVE_FIRST_VALUE_IN_INDIRECT_DRAW
-        );
+        let downlevel_support = adapter
+            .get_downlevel_capabilities()
+            .flags
+            .contains(DownlevelFlags::COMPUTE_SHADERS);
```

Comment:
Why was the other flag removed?

Reply:
https://github.com/gfx-rs/wgpu/pull/7535